ipam/ipam/main.go

```diff
@@ -44,7 +44,7 @@ func main() {
 		fmt.Println("Schema created.")
 		return
 	}
-
+	cred := common.MakeCredentialFromArgs(username, password)
```

Comment:
Where does 'username' and 'password' come from here? I don't see it being extracted from the command line?